metadata-ingestion/src/datahub/configuration/pydantic_field_deprecation.py

```diff
@@ -7,16 +7,14 @@
 from datahub.utilities.global_warning_util import add_global_warning
 
 
-def pydantic_field_deprecated(
```

Comment:
Okay, so the responsibility of letting user know about the alternatives of deprecated field is entirely on the message field.

Reply:
yes. if you have an alternate field, use the `pydantic_renamed_field` instead

Reply:
makes sense.